torch/onnx/_internal/fx/export.py

```diff
@@ -15,11 +17,120 @@
 from torch.fx.experimental import proxy_tensor
 from torch.fx.passes import fake_tensor_prop
 from torch.nn.utils import stateless
+from torch.onnx import _type_utils
 from torch.onnx._globals import GLOBALS as ONNX_GLOBALS
-from torch.onnx._internal import jit_utils, registration
 from torch.utils import _pytree
 
 
+# A simple lookup table for atenlib functions
+_ATENLIB_FUNCTIONS = {
```

Comment:
You can auto-generate this table by, e.g.,
```python
for op in dir(ops.core):
  if instance(op, ONNXFunction):
    _ATENLIB_FUNCTIONS[op.name] = op
```

Reply:
I would like to make it explicit for now and avoid traversing a module, as we used to do it in torch.onnx symbolic functions and it proved to be a headache. When we have a proper dispatcher we can switch to use that